webapp/channels/src/components/channel_name_form_field/channel_name_form_field.tsx

```diff
@@ -14,7 +14,7 @@ import type {CustomMessageInputType} from 'components/widgets/inputs/input/input
 import Input from 'components/widgets/inputs/input/input';
 import URLInput from 'components/widgets/inputs/url_input/url_input';
 
-import Constants from 'utils/constants';
+import {Constants} from 'utils/constants';
```

Comment:
Was this done automatically? I was going to suggest removing the default export from `utils/constants` to make us consistent here, but it looks like we use `import Constants from 'utils/constants'` all over the place.

I'm 0/5 on actually changing this elsewhere though since I don't think this inconsistency is a problem, and if we're spending any real energy on cleaning up imports to this file, I think it'd be better to spend that time breaking up the file into smaller pieces to help code splitting